GeeksCoreLibrary/Modules/PostalServices/PostNL/Services/PostNLService.cs

```diff
@@ -0,0 +1,400 @@
+                    foreach (WiserItemModel orderLine in orderLines)
+                    {
+                        postNlRequest.Shipments.First()
+                            .Customs.Content.Add(new CustomsContentModel
+                            {
+                                Description = orderLine.GetDetailValue("title"),
+                                CountryOfOrigin = "NL",
+                                HSTariffNr = "621112",
+                                Quantity = orderLine.GetDetailValue("quantity"),
+                                Value = orderLine.GetDetailValue("price").Replace(",", "."),
+                                Weight = "500"
+                            });
+                    }
+                }
+
+                var postNlResponse = await CreateTrackTraceLabelAsync(orderId, postNlRequest);
+                if (postNlResponse?.ResponseShipments == null || !postNlResponse.ResponseShipments.Any())
+                {
+                    result.Add($"Order {orderId}: Er is iets fout gegaan met de koppeling met de PostNL API.");
+                    continue;
+                }
+
+                barcode = postNlResponse.ResponseShipments.First().Barcode;
+
+                orderDetails.SetDetail("postnl_barcode", barcode);
+                orderDetails.SetDetail("country_code", await GetCountryCodeAsync(orderDetails, "country"));
+
+                await wiserItemsService.UpdateAsync(orderDetails.Id, orderDetails);
+                foreach (var labelResponseShipment in postNlResponse.ResponseShipments)
+                {
+                    if (labelResponseShipment.Errors.Any())
+                    {
+                        result.Add($"Order {orderId}: De PostNL API heeft een of meer fouten gegeven: {String.Join(", ", labelResponseShipment.Errors.Select(x => x.Description))}");
+                        continue;
+                    }
+
+                    foreach (var label in labelResponseShipment.Labels)
+                    {
+                        await wiserItemsService.AddItemFileAsync(new WiserItemFileModel
+                        {
+                            ItemId = ulong.Parse(orderId),
+                            Content = Convert.FromBase64String(label.Content),
+                            Extension = ".pdf",
+                            FileName = $"{label.Labeltype}.pdf",
+                            ContentType = "application/pdf",
+                            PropertyName = "postnl_label",
+                            Title = barcode
+                        });
+                    }
+                }
+
+                result.Add($"Order {orderId}: Er is succesvol een verzendlabel gegenereerd en verstuurd naar de klant, deze kan gevonden worden op de tab 'PostNL' van deze order. De track&trace code is: {barcode}");
+            }
+
+            return $"<ul><li>{String.Join("</li><li>", result)}</li></ul>";
```

Comment:
Dit was me eerder niet opgevallen, maar de HTML die hier teruggegeven wordt, was specifiek voor de Wiser inrichting van Beachlife gemaakt. Dit is niet handig om zo in de GCL te laten staan. Deze functie zou gewoon de variabele `result` direct moeten teruggeven, zonder daar HTML van te maken. Kun je dat aanpassen?

Reply:
@alexxxx1992 Je bent deze nog vergeten aan te passen in je laatste commit. Als je dit nog wilt aanpassen, dan keur ik hem daarna goed.